src/Smd.php

```diff
@@ -322,7 +332,7 @@ public function addService($service)
      * @param  array $services
      * @return self
      */
-    public function addServices(array $services)
+    public function addServices(array $services) : self
```

Comment:
In general for cases like that we want to use something more generic than `array`, probably `iterable`.

Reply:
Good point. I would like to open an issue for that to handle it separately. Are you okay with that?